0_intro_to_jax_for_ml.ipynb

```diff
@@ -0,0 +1,2081 @@
+{
```

Comment:
<p>You seem to be giving the cprrect answer here by default?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='89'/>

Reply:
<p>This has changed now. </p>